index.html

```diff
@@ -47,4 +47,87 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
 <button id="submit" type="submit">Submit</button>
 </form>
 </body>
-</html>
\ No newline at end of file
+</html> -->
+
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
- [ ] Good job so far but I think it would be better to include an explanation, **(a "why")**, for this requested change. 😎😎😎😎

 _Remember, when you provide a reason or explanation for your feedback, it helps the recipient to understand the context and importance of your comments. It also helps to establish a clear connection between the feedback and the goals or objectives that the recipient is trying to achieve. 👍_